eval/final/components.py

```diff
@@ -480,3 +480,36 @@ def find_node_dataset_directories(base_dir: str):
 
     with open(mt_bench_branch_output.path, "w", encoding="utf-8") as f:
         json.dump(mt_bench_branch_data, f, indent=4)
+
+
+@component(base_image=PYTHON_IMAGE, install_kfp_package=False)
+def generate_metrics_report_op(
+    mmlu_branch_output: Input[Artifact],
+    mt_bench_branch_output: Input[Artifact],
+    mt_bench_output: Input[Artifact],
```

Comment:
WDYT about making these inputs being read from the PVC instead of them being input artifact (as the other steps do)? It would make things faster (and more uniform).

Other option would be to emit those metrics directly from `final_eval_task` and `output_mt_bench_task` steps. I think that would be the best option, WDYT?

Reply:
Emitting them from `final_eval_task` and `output_mt_bench_task` was my first approach, but it has some bad consequences for the UX, specifically metrics from each step get rendered in their own sub-column per run column. So it becomes difficult to read, and if we want to add more metrics from other steps the problem will get worse. So I think keeping a single step that collects all the metrics we want to report out is the best approach for now, as it keeps everything ins a single column and makes comparing run metrics easier.

Reply:
>WDYT about making these inputs being read from the PVC instead of them being input artifact (as the other steps do)? It would make things faster (and more uniform).

Great idea 😄  I will make that change, Thanks!